src/components/Hero.astro

```diff
@@ -1,4 +1,32 @@
-<div class="mx-auto max-w-lg pt-10">
-  <h1 class="text-6xl">DeltaHacks 10</h1>
-  <p>Back and better than ever!</p>
+<div
+  class="flex h-screen flex-col justify-between bg-blue-200 bg-[url('/beachBackground.svg')] p-4 pt-24 sm:justify-normal sm:gap-24 sm:bg-cover sm:p-24"
+>
+  <div>
+    <h1 class="text-5xl font-extrabold tracking-tight md:text-8xl lg:text-9xl">
+      DELTAHACKS<span class="text-rose-600">X</span>
+    </h1>
+    <!-- RGBA(231, 34, 80, 1) -->
+    <h3 class="text-xl font-semibold leading-6 tracking-tight text-cyan-900">
```

Comment:
Please use `<p class="tagline">` instead of `<h3>`, as the subtitle is not a heading

https://mikebifulco.com/posts/semantic-html-heading-subtitle